ui-tests/src/test/java/org/openmrs/reference/AddDiagnosisToVisitNoteTest.java

```diff
@@ -33,12 +32,12 @@
     @Before
     public void setup() {
         patient = createTestPatient();
-        createTestVisit();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+    @Test
     @Category(BuildTests.class)
-    public void AddDiagnosisToVisitNoteTest() {
```

Comment:
Did you confirm whether the naming convention has not been used somewhere  please , otherwise we want to take ticket per ticket now

Reply:
Thanks @sherrif10 for this! 
This is a method name which is used within the test class, no any other method bears this name. By java conversation method names should begin with small letters and this compelled me to make things right since we are the QA Engineers working on the community project